src/lib.rs

```diff
@@ -1,31 +1,46 @@
 /*!
 # Unicode Blocks
 
-This crate contains a list of all unicode blocks and provides some functions to search across them.
+This crate contains a list of all unicode blocks and provides some functions
+to search across them.
+
+## Features
+
+* `all` – Include an array that contains references to all unicode blocks.
+
+  As this takes quite a bit of memory (approx 2.6k on a 64bit system) this is
+  turned off by default.
 
 ## Examples
 
 #### Given a character, determine what unicode block contains it.
 
-```rust
-assert_eq!(unicode_blocks::BASIC_LATIN, unicode_blocks::find_unicode_block('A').unwrap());
+```
```

Comment:
I prefer using ```` ```rust ````.

Reply:
Because I copy doc comments in `lib.rs` and paste them to `README.md`. I don't want to omit the language name for syntax highlighting.